app/models/webhooks/slack/motion_closed.rb

```diff
@@ -15,4 +15,8 @@ def attachment_text
   def attachment_fields
     [view_motion_on_loomio]
   end
+
+  def attachment_color
```

Comment:
Thanks for all the feedback! We moved the empty attachment color up to `base.rb`, and fixed a minor syntax error with "yes" and "no" in `en.yml` this time around (needed quotation marks).